iotilecore/iotile/core/hw/virtual/__init__.py

```diff
@@ -7,11 +7,15 @@
                            RPCNotFoundError, RPCInvalidArgumentsError,
                            RPCInvalidReturnValueError, TileNotFoundError,
                            RPCErrorCode, BusyRPCResponse, unpack_rpc_payload, pack_rpc_payload,
-                           pack_rpc_response)
+                           pack_rpc_response, unpack_rpc_response,
+                           VALID_RPC_EXCEPTIONS)
+
 from .virtualinterface import VirtualIOTileInterface
 
 __all__ = ['VirtualTile', 'VirtualIOTileDevice', 'tile_rpc',
```

Comment:
It seems like we're collecting quite the list of imports here. Is this a normal thing or do alternative solutions exist for situations like this? I tried briefly to see if anyone posted thoughts about this on the web without success. 

Not really important, just a thought.

Reply:
Yeah we should tag this for a refactor to see if these are all still required or if there is a way we can group some higher level functionality that hides some of them.

Reply:
Tagged as #771